### PR TITLE
PYIC-3028 Added env variable CIMIT_GET_CONTRAINDICATORS_LAMBDA_ARN

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
- Add env variable CIMIT_GET_CONTRAINDICATORS_LAMBDA_ARN 
- Update cimit account id for new dev
### What changed
Added env variable CIMIT_GET_CONTRAINDICATORS_LAMBDA_ARN and updated limit account id for new dev env.

### Issue tracking

- [PYIC-3028](https://govukverify.atlassian.net/browse/PYIC-3028)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3028]: https://govukverify.atlassian.net/browse/PYIC-3028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ Mappings:
       asyncCriLambdaTimeout: 30
     "175872367215": # New Development Account
       provisionedConcurrency: 0
-      ciStorageAccountId: 158853307826 # new CIMIT dev
+      ciStorageAccountId: 175872367215 # new CIMIT dev
```
Comment (therealslimsladey) [inline deploy/template.yaml]: This should be in a separate PR once the dev CIMIT stubs have been deployed

File: deploy/template.yaml
Comment (therealslimsladey) [inline deploy/template.yaml:1]: Will also need the InvokeFunction permission for the getContraIndicatorCredential lambda

**Review by TalatAydinGDS — DISMISSED** [review]
LGTM

File: deploy/template.yaml [hunk 2]
```diff
@@ -1534,6 +1534,19 @@ Resources:
                   - EnvironmentConfiguration
                   - !Ref AWS::AccountId
                   - environment
+          CIMIT_GET_CONTRAINDICATORS_LAMBDA_ARN: !Sub
+            - "arn:aws:lambda:eu-west-2:${contra_indicator_storage_account_id}:function:getContraIndicatorCredential-${env}"
+            - contra_indicator_storage_account_id: !FindInMap
```
Comment (therealslimsladey) [inline deploy/template.yaml]: This should use the same account if `UseIndividualCiMitStubs` is set, same as for `CI_STORAGE_GET_LAMBDA_ARN` contra_indicator_storage_account_id`:
```
            - contra_indicator_storage_account_id: !If
                - UseIndividualCiMitStubs
                - !Ref AWS::AccountId
                - !FindInMap
                  - EnvironmentConfiguration
                  - !Ref AWS::AccountId
                  - ciStorageAccountId
```

File: deploy/template.yaml [hunk 3]
```diff
@@ -1860,6 +1895,19 @@ Resources:
...
```
Comment (therealslimsladey) [inline deploy/template.yaml]: This should use the same account if `UseIndividualCiMitStubs` is set, same as for `CI_STORAGE_GET_LAMBDA_ARN` contra_indicator_storage_account_id`:
```
            - contra_indicator_storage_account_id: !If
                - UseIndividualCiMitStubs
                - !Ref AWS::AccountId
                - !FindInMap
                  - EnvironmentConfiguration
                  - !Ref AWS::AccountId
                  - ciStorageAccountId
```

File: deploy/template.yaml [hunk 4]
```diff
@@ -1600,6 +1613,28 @@ Resources:
                         - EnvironmentConfiguration
                         - !Ref AWS::AccountId
                         - environment
+        - Statement:
+            - Sid: invokeGetCiFunction
```
Comment (therealslimsladey) [inline deploy/template.yaml]: Sid should be something like `invokeGetContraIndicatorCredentialFunction`

File: deploy/template.yaml [hunk 5]
```diff
@@ -1925,6 +1973,28 @@ Resources:
...
```
Comment (therealslimsladey) [inline deploy/template.yaml]: Sid should be something like `invokeGetContraIndicatorCredentialFunction`

**Review by therealslimsladey — CHANGES_REQUESTED** [review]

File: deploy/template.yaml (hunk 2 above)
Comment (rpayal) [reply deploy/template.yaml]: done

File: deploy/template.yaml (hunk 3 above)
Comment (rpayal) [reply deploy/template.yaml]: done

File: deploy/template.yaml (hunk 4 above)
Comment (rpayal) [reply deploy/template.yaml]: done

File: deploy/template.yaml (hunk 5 above)
Comment (rpayal) [reply deploy/template.yaml]: done

**Review by therealslimsladey — APPROVED** [review]
